oidc/client.go

```diff
@@ -763,7 +763,7 @@ func (c *Client) RefreshToken(refreshToken string) (jose.JWT, error) {
 		return jose.JWT{}, err
 	}
 
-	jwt, err := jose.ParseJWT(t.IDToken)
+	jwt, err := jose.ParseJWT(t.AccessToken)
```

Comment:
> AccessTokens are opaque. Why would it be safe to assume they're a JWT?
  
I don't see my initial question address.

I get what you're requesting: a flow for when an provider doesn't return an ID Token as part of the refresh response. I'm confused why this would be the solution.